packages/types/config/runtime.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * NuxtConfigurationRuntimeConfig
+ * Documentation : TBD
+ * NuxtRuntimeConfig interface can be extended by users to enable intellisense on $config
+ */
+
+export interface NuxtRuntimeConfig {
+  [key: string]: any
+}
+
+export type NuxtConfigurationRuntimeConfig = NuxtRuntimeConfig | ((env: Record<any, string>) => NuxtRuntimeConfig)
```

Comment:
You can use `NuxtConfigurationEnv` in place of `Record<any, string>` right?

Reply:
As mentioned, `NuxtConfigurationEnv` is getting deprecated. Same type but they are different.

Reply:
But then the key should be `string` then?
I can't imagine that things will work if we pass `Symbol()` as key for example as that is not serializable.

Reply:
Indeed keys should be string only.

`@types/node/global.d.ts`:

```ts
    interface Dict<T> {
        [key: string]: T | undefined;
    }

interface ProcessEnv extends Dict<string> {}
```

We may also directly use `ProcessEnv` type because object loaded from `.env` is same type

Reply:
@rchl That's my bad bor this one, should be `string, string` indeed